package.json

```diff
@@ -2,10 +2,11 @@
   "name": "meriyah",
   "version": "4.1.1",
   "description": "A 100% compliant, self-hosted javascript parser with high focus on both performance and stability",
-  "main": "dist/meriyah.umd.js",
+  "main": "dist/meriyah.umd.cjs",
```

Comment:
This change is not necessary. For nodejs supports `exports.import/require`, it will ignore main field.
For old nodejs doesn't support `exports.import/require`, this main field works fine.

I tested current release with nodejs v10 (doesn't support `exports.import/require`), v12, v14, all worked fine.

Reply:
I think you mistook this file as cjs, the real cjs file is `meriyah.cjs.js`.

The `.cjs` in `*.umd.cjs` is just for making nodejs happy because this package is marked as `"type": "module"`, nodejs (v12+) thinks `meriyah.umd.js` should be an esm file, so we added `.cjs` to calm down nodejs.

Reply:
You can update both "main" and "exports.require" to `meriyah.cjs`, also need to update rollup setup to change the file name from `meriyah.cjs.js` to `meriyah.cjs`.

Reply:
done